alexandrie/src/config/mod.rs

```diff
@@ -30,12 +30,19 @@ pub struct GeneralConfig {
     /// The port to listen on.
     pub port: u16,
 }
-
+/// The asset confiuration options struct
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct AssetCoonfig {
```

Comment:
Did you mean `AssetConfig` here ?  
I would additionally suggest to add a `s` to `Asset`, since there is more than one asset used.  
```suggestion
pub struct AssetsConfig {
```